### PR TITLE
[Misc] Fix minor issues

### PR DESCRIPTION
**Pull request by Eideren**
# PR Details
See changes, those were reported through the build warnings and seemed significant enough to deal with.

## Types of changes
- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: sources/tools/Stride.TextureConverter/Backend/Wrappers/FINetWrapper/FreeImageWrapper.cs [hunk 1]
```diff
@@ -2060,7 +2060,7 @@ public static FIMULTIBITMAP OpenMultiBitmapFromStream(Stream stream, ref FREE_IM
 				if (!mdib.IsNull)
 					CloseMultiBitmap(mdib, FREE_IMAGE_SAVE_FLAGS.DEFAULT);
 
-				if (handle != null)
+				if (handle.IsNull == false)
```
Comment (manio143) [inline sources/tools/Stride.TextureConverter/Backend/Wrappers/FINetWrapper/FreeImageWrapper.cs:2063]: Any reason why you're not using `if (!handle.IsNull)`?

Comment (Eideren) [reply sources/tools/Stride.TextureConverter/Backend/Wrappers/FINetWrapper/FreeImageWrapper.cs:2063]: Nothing besides my own style, I don't tend to use it as I find that character quite easy to miss when skimming through logic

**Review by manio143 — APPROVED** [review]

**Comment by VaclavElias** [comment]
I saw somewhere references to .NET 5 in the master branch, somewhere in hosting when the project is built in the output window. I have take a screenshot or get output for more details..

**Comment by VaclavElias** [comment]
@Eideren Here it is. Note, I don't have the most recent version.

38>"C:\Users\Vacla\AppData\Local\Temp\Stride\AssemblyProcessor\7D51791B4987377D755D7ADE3C430DED0890CCA99053F1FF77B0CD00B92F51C7\netstandard2.0\Stride.Core.AssemblyProcessor.Packed.dll" --platform=Windows --targetFramework=v6.0 --auto-module-initializer --serialization --assembly="D:\Projects\GitHub\stride\build\..\Bin\\Stride.Core.dll" --references-file="obj\Debug\net6.0-windows7.0\StrideReferences.cache" --Werror "obj\Debug\net6.0-windows7.0\Stride.Core.Assets.dll"
38>Patch for assembly [Stride.Core.Assets, Version=4.1.0.1, Culture=neutral, PublicKeyToken=null]
28>Done building project "Stride.Core.Design.Tests.csproj".
39>Stride.Code.Tests -> D:\Projects\GitHub\stride\sources\tools\Stride.Code.Tests\bin\Debug\net6.0-windows7.0\Stride.Code.Tests.dll
17>   Creating library D:\Projects\GitHub\stride\sources\engine\Stride.Native\bin\Debug\net6.0\\win-x64\libstride.lib and object D:\Projects\GitHub\stride\sources\engine\Stride.Native\bin\Debug\net6.0\\win-x64\libstride.exp
38>Stride.Core.Assets -> D:\Projects\GitHub\stride\sources\assets\Stride.Core.Assets\bin\Debug\net6.0-windows7.0\Stride.Core.Assets.dll
17>WindowsDesktop.vcxproj -> D:\Projects\GitHub\stride\sources\engine\Stride.Native\bin\Debug\net6.0\win-x64\libstride.dll
17>"C:\Users\Vacla\AppData\Local\Temp\Stride\AssemblyProcessor\7D51791B4987377D755D7ADE3C430DED0890CCA99053F1FF77B0CD00B92F51C7\netstandard2.0\Stride.Core.AssemblyProcessor.Packed.dll" --platform=Windows --targetFramework=v6.0 --serialization --parameter-key --assembly="D:\Projects\GitHub\stride\build\..\Bin\\Stride.Core.dll" --references-file="obj\Debug\net6.0\StrideReferences.cache" --Werror "obj\Debug\net6.0\Stride.Native.dll"
38>D:\Projects\GitHub\stride\sources\assets\Stride.Core.Assets\Stride.Core.Assets.csproj : warning NU1603: Microsoft.NET.HostModel 5.0.0-preview.1.20120.5 depends on System.Text.Json (>= 5.0.0-alpha.1.19563.6) but **System.Text.Json 5.0.0-alpha.1.19563.6 was not found. An approximate best match of System.Text.Json 5.0.0-preview.1.20120.5 was resolved.**
40>Stride.Core.ProjectTemplating.Tests -> D:\Projects\GitHub\stride\sources\tools\Stride.Core.ProjectTemplating.Tests\bin\Debug\net6.0-windows7.0\win7-x64\Stride.Core.ProjectTemplating.Tests.dll
17>Patch for assembly [Stride.Native, Version=4.1.0.1, Culture=neutral, PublicKeyToken=null]
17>Stride.Native -> D:\Projects\GitHub\stride\sources\engine\Stride.Native\bin\Debug\net6.0\Stride.Native.dll
17>C:\Program Files\dotnet\sdk\6.0.101\Sdks\NuGet.Build.Tasks.Pack\buildCrossTargeting\NuGet.Build.Tasks.Pack.targets(221,5): warning NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
17>Successfully created package 'D:\Projects\GitHub\stride\sources\targets\..\..\bin\packages\Stride.Native.4.1.0.1-beta.nupkg'.
17>Done building project "Stride.Native.csproj".
41>------ Rebuild All started: Project: Stride.Audio, Configuration: Debug Any CPU ------
38>Done building project "Stride.Core.Assets.csproj".
35>Patch for assembly [Stride.Core.Quantum.Tests, Version=4.1.0.1, Culture=neutral, PublicKeyToken=null]
38>C:\Program Files\dotnet\sdk\6.0.101\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(221,5): warning NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
38>Successfully created package 'D:\Projects\GitHub\stride\sources\targets\..\..\bin\packages\Stride.Core.Assets.4.1.0.1-beta.nupkg'.
38>Done building project "Stride.Core.Assets.csproj".
42>------ Rebuild All started: Project: Stride.Core.Tasks, Configuration: Debug Any CPU ------
43>------ Rebuild All started: Project: Stride.Core.Assets.Quantum, Configuration: Debug Any CPU ------
44>------ Rebuild All started: Project: Stride.Core.Assets.Tests, Configuration: Debug Any CPU ------
43>D:\Projects\GitHub\stride\sources\assets\Stride.Core.Assets.Quantum\Stride.Core.Assets.Quantum.csproj : warning NU1603: Microsoft.NET.HostModel 5.0.0-preview.1.20120.5 depends on System.Text.Json (>= 5.0.0-alpha.1.19563.6) but System.Text.Json 5.0.0-alpha.1.19563.6 was not found. An approximate best match of System.Text.Json 5.0.0-preview.1.20120.5 was resolved.
44>D:\Projects\GitHub\stride\sources\assets\Stride.Core.Assets.Tests\Stride.Core.Assets.Tests.csproj : warning NU1603: Microsoft.NET.HostModel 5.0.0-preview.1.20120.5 depends on System.Text.Json (>= 5.0.0-alpha.1.19563.6) but System.Text.Json 5.0.0-alpha.1.19563.6 was not found. An approximate best match of System.Text.Json 5.0.0-preview.1.20120.5 was resolved.
42>D:\Projects\GitHub\stride\sources\core\Stride.Core.Tasks\Stride.Core.Tasks.csproj : warning NU1603: Microsoft.NET.HostModel 5.0.0-preview.1.20120.5 depends on System.Text.Json (>= 5.0.0-alpha.1.19563.6) but System.Text.Json 5.0.0-alpha.1.19563.6 was not found. An approximate best match of System.Text.Json 5.0.0-preview.1.20120.5 was resolved.
35>Stride.Core.Quantum.Tests -> D:\Projects\GitHub\stride\bin\Tests\Stride.Core.Quantum.Tests\Windows\Stride.Core.Quantum.Tests.dll

**Comment by VaclavElias** [comment]
And another one when I build Stride.Games project

Build started...
1>------ Build started: Project: Stride.Core.Tasks, Configuration: Debug Any CPU ------
1>D:\Projects\GitHub\stride\sources\core\Stride.Core.Tasks\Stride.Core.Tasks.csproj : warning NU1603: Microsoft.NET.HostModel 5.0.0-preview.1.20120.5 depends on System.Text.Json (>= 5.0.0-alpha.1.19563.6) but System.Text.Json 5.0.0-alpha.1.19563.6 was not found. An approximate best match of System.Text.Json 5.0.0-preview.1.20120.5 was resolved.
1>D:\Projects\GitHub\stride\sources\assets\Stride.Core.Assets\Stride.Core.Assets.csproj : warning NU1603: Microsoft.NET.HostModel 5.0.0-preview.1.20120.5 depends on System.Text.Json (>= 5.0.0-alpha.1.19563.6) but System.Text.Json 5.0.0-alpha.1.19563.6 was not found. An approximate best match of System.Text.Json 5.0.0-preview.1.20120.5 was resolved.
1>Done building project "Stride.Core.Packages.csproj".
1>Stride.Core.Tasks -> D:\Projects\GitHub\stride\sources\core\Stride.Core.Tasks\bin\Debug\net6.0-windows7.0\Stride.Core.Tasks.dll
1>D:\Projects\GitHub\stride\sources\core\Stride.Core.Tasks\Stride.Core.Tasks.csproj : warning NU1603: Microsoft.NET.HostModel 5.0.0-preview.1.20120.5 depends on System.Text.Json (>= 5.0.0-alpha.1.19563.6) but System.Text.Json 5.0.0-alpha.1.19563.6 was not found. An approximate best match of System.Text.Json 5.0.0-preview.1.20120.5 was resolved.
1>Done building project "Stride.Core.Tasks.csproj".
1>D:\Projects\GitHub\stride\sources\core\Stride.Core.Tasks\Stride.Core.Tasks.csproj : warning NU1603: Microsoft.NET.HostModel 5.0.0-preview.1.20120.5 depends on System.Text.Json (>= 5.0.0-alpha.1.19563.6) but System.Text.Json 5.0.0-alpha.1.19563.6 was not found. An approximate best match of System.Text.Json 5.0.0-preview.1.20120.5 was resolved.
1>Done building project "Stride.Core.Tasks.csproj".
1>Done building project "Stride.Core.Tasks.csproj".
========== Build: 1 succeeded, 0 failed, 16 up-to-date, 0 skipped ==========

**Comment by Eideren** [comment]
@VaclavElias try PR #1276 out, that one updates a couple of packages, I don't have those warning when building with it

**Comment by VaclavElias** [comment]
I tried both #1276, and also master branch and those .NET 5 references are gone. So this was a false alert. All seems ok :)

**Pull request merged**